docs/guides/administrator/adding-and-managing-tenants.md

```diff
@@ -0,0 +1,144 @@
+If you have created your cluster with [authentication disabled](https://opstrace.com/docs/references/cluster-configuration#data_api_authentication_disabled), option you can skip to the [add a new tenant with the UI](adding-and-managing-tenants.md#add-a-new-tenant-with-the-ui) section.
+
+### 1) Create a new RSA Key Pair
+
+Create a new RSA key pair and store it in a file with this command:
+
+```bash
+./opstrace ta-create-keypair ./custom-keypair.pem
+```
+
+Note: The `ta-` prefix represents the idea of "tenant API authentication." All
+`ta-*` commands offered by the Opstrace CLI are new and should be thought of as
+experimental (command names and signatures are subject to potentially big
+changes in the future).
+
+After running this command, you have a local file `./custom-keypair.pem` in your
+file system, with locked-down file permissions.
+It is important to understand
+that this file contains a secret, the _private_ key.
+
+### 2) Create the Token
+
+The following command creates a new authentication token, signed with the
+private key of the key pair generated in the first step:
+
+```bash
+./opstrace ta-create-token showdown newtenant custom-keypair.pem > token-showdown-newtenant.jwt
+```
+
+The token (emitted via `stdout` and captured in the file
+`token-showdown-newtenant.jwt`) is a standards-compliant JSON Web Token (JWT),
+implementing Opstrace-specific conventions:
+
+* in the JWT header, it encodes an ID of the public key via which it can be
+  cryptographically validated.
+* in the JWT payload section, it encodes the name of the associated tenant.
+
+For this token to become useful, an Opstrace instance needs to be configured
+with the _public key_ corresponding to the private key that was used to sign the
+token.
+Let's do that.
+
+### 3) Add the Public Key to a Running Opstrace Instance
+
+You can think of this step as adding a new trust anchor to the trust store of a
+running Opstrace instance.
+Just like with X.509 certificates, trusting a public
+key means that authentication proof signed with the correspond _private_ key is
+accepted (reminder: in
+[public-key cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography),
+verification of authentication proof only ever needs the non-sensitive public
+key material and not the private key data—that's the beauty).
+
+So, to make an existing Opstrace instance _trust_ the authentication token
+generated in the previous step, we have to put the public key into the instance:
+
+```bash
+./opstrace ta-pubkeys-add aws showdown custom-keypair.pem
+```
+
+## Add a New Tenant in the UI
+
+Visit `https://showdown.opstrace.io/cluster/tenants` and press the `Add Tenant` button.
+Type the name of the new tenant (here: `newtenant`).
+
+![add tenant gif](../../assets/tenants-guide-add.gif)
+
+**Warning**: we do not yet do strict tenant name validation.
+To make sure things
+work, please keep the name lower case `[a-z]` for now.
```

Comment:
CC @MatApple @terrcin and the corresponding bug ticket: https://github.com/opstrace/opstrace/issues/733 -- let's address this soon and then remove this bit from this documentation.